README.md

```diff
@@ -14,38 +14,20 @@ _This project was previously called [Cleopatra]._
 
 > Visit [perf-html.io](https://perf-html.io/) :rocket:
 
-This project is live on [https://perf-html.io/](https://perf-html.io/). You need to install the add-on provided at that website to record profile data from Firefox.
+This project is live on [https://perf-html.io/](https://perf-html.io/). The website includes instructions on how to get going to start recording and viewing performance profiles.
 
 ### Development
 
-> You'll find documentation for the project in the [docs folder](./docs) as well as `README.md` files in most folders in [/src](./src).
+Want some more information about contributing and running perf.html locally? We have plenty of docs to get you started.
 
-We use [Yarn](http://yarnpkg.com/) as our dependency manager so you'll need to
-go and install it before running the following commands.
-
-```bash
```

Comment:
We've found with the debugger that more experienced users really want this quick setup set of commands they can copy and paste into the console.  We just also link to the annotated version you have in the contributing guide.